.config/cargo_spellcheck.dic

```diff
@@ -0,0 +1,52 @@
+45
+
+WebAssembly
+Wasm
+validator
+ERC
+fuzzer
+SHA
+BLAKE2
+KECCAK
+ABI
+FFI
+UI
+crypto
+deploy
+layed
+dispatchable
+SRML
+encodable
+decodable
+spreaded
+callee
+pre
+defrag
+defragments
```

Comment:
```suggestion
defragment/SD
defragmentation
```

I'd recommend to take a look into how affix files work, it's a bit painful but worth it - allows to generalize a few patterns :+1: